app/createTournament/createTournamentDialog.js

```diff
@@ -0,0 +1,25 @@
+import * as tournamentValidation from './createTournamentBusiness.js';
```

Comment:
This is aligned with my previous comment. Be careful, if this is only validation, rename the js and check out it's only validation.